packages/snap-preact-components/src/types.ts

```diff
@@ -38,6 +38,9 @@ export type BreakpointsProps = {
 
 export type BreakpointsEntry = {
 	[property: string]: any;
+	slidesPerView?: number;
+	slidesPerGroup?: number;
+	spaceBetween?: number;
```

Comment:
This makes this type specific to carousel, previous typing should have supported any number of properties. Was this necessary?